tests/test_inclined_exponential.py

```diff
@@ -126,211 +281,223 @@ def test_edge_on():
+                               str(gsparams.folding_threshold) + "\nTotal flux = " +
+                               str(total_flux) + "\nContained flux = " + str(contained_flux) +
+                               "\nLost = " + str((total_flux-contained_flux)/(total_flux)))
 
 @timer
 def test_eq_ne():
     """ Check that equality/inequality works as expected."""
     gsp = galsim.GSParams(folding_threshold=1.1e-3)
-
-    # First test that some different initializations that should be equivalent:
-    gals = [galsim.InclinedExponential(0.1*galsim.radians, 3.0),
-            galsim.InclinedExponential(0.1*galsim.radians, 3.0, 0.3),  # default h/r = 0.1
-            galsim.InclinedExponential(0.1*galsim.radians, 3.0, scale_height=0.3),
-            galsim.InclinedExponential(0.1*galsim.radians, 3.0, scale_h_over_r=0.1),
-            galsim.InclinedExponential(0.1*galsim.radians, 3.0, flux=1.0),  # default flux=1
-            galsim.InclinedExponential(-0.1*galsim.radians, 3.0),  # negative i is equivalent
-            galsim.InclinedExponential((np.pi--0.1)*galsim.radians, 3.0),  # also pi-theta
-            galsim.InclinedExponential(18./np.pi*galsim.degrees, 3.0),
-            galsim.InclinedExponential(inclination=0.1*galsim.radians, scale_radius=3.0,
-                                       scale_height=0.3, flux=1.0),
-            galsim.InclinedExponential(flux=1.0, scale_radius=3.0,
-                                       scale_height=0.3, inclination=0.1*galsim.radians)]
-
-    for gal in gals[1:]:
-        print(gal)
-        gsobject_compare(gal, gals[0])
-
-    gals = [galsim.InclinedExponential(0.1*galsim.radians, 3.0, 0.3),
-            galsim.InclinedExponential(0.1*galsim.degrees, 3.0, 0.3),
-            galsim.InclinedExponential(0.1*galsim.degrees, 3.0, scale_h_over_r=0.2),
-            galsim.InclinedExponential(0.1*galsim.radians, 3.0, 3.0),
-            galsim.InclinedExponential(0.2*galsim.radians, 3.0, 0.3),
-            galsim.InclinedExponential(0.1*galsim.radians, 3.1, 0.3),
-            galsim.InclinedExponential(0.1*galsim.radians, 3.1),
-            galsim.InclinedExponential(0.1*galsim.radians, 3.0, 0.3, flux=0.5),
-            galsim.InclinedExponential(0.1*galsim.radians, 3.0, 0.3, gsparams=gsp)]
-    all_obj_diff(gals)
+    
+    for mode in ("InclinedExponential","InclinedSersic"):
+
+        # First test that some different initializations that should be equivalent:
+        same_gals = [get_prof(mode,0.1*galsim.radians, 3.0),
+                get_prof(mode,0.1*galsim.radians, 3.0, 0.3),  # default h/r = 0.1
+                get_prof(mode,0.1*galsim.radians, 3.0, scale_height=0.3),
+                get_prof(mode,0.1*galsim.radians, 3.0, scale_h_over_r=0.1),
+                get_prof(mode,0.1*galsim.radians, 3.0, flux=1.0),  # default flux=1
+                get_prof(mode,-0.1*galsim.radians, 3.0),  # negative i is equivalent
+                get_prof(mode,(np.pi--0.1)*galsim.radians, 3.0),  # also pi-theta
```

Comment:
Looks like this was an error previously, but this was supposed to be `np.pi-0.1` I think.